readme.md

```diff
@@ -18,6 +18,9 @@ var isAbsoluteUrl = require('is-absolute-url');
 isAbsoluteUrl('http://sindresorhus.com/foo/bar');
 //=> true
 
+isAbsoluteUrl('//sindresorhus.com');
+//=> false
```

Comment:
Yeah, I think it's should link to an explanation of why it's not an absolute path.